tools/hooks/20-flake8.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+echo "Executing 'flake8' formatter..."
+
+pushd $PROJECT_ROOT > /dev/null
+  python -m flake8        \
+    --count               \
+    --exit-zero           \
+    --max-complexity=10   \
```

Comment:
Remove max complexity. Otherwise I get this garbage:
```
Importing ./tools/hooks/20-flake8.sh...
Executing 'flake8' formatter...
./armory/configuration.py:16:1: C901 'validate_config' is too complex (11)
def validate_config(config: dict) -> None:
^
./armory/__main__.py:287:1: C901 'run' is too complex (13)
def run(command_args, prog, description) -> int:
^
./armory/baseline_models/pytorch/ucf101_mars.py:30:1: C901 'preprocessing_fn_torch' is too complex (20)
def preprocessing_fn_torch(
^
./armory/metrics/task.py:413:1: C901 'word_error_rate' is too complex (12)
def word_error_rate(y, y_pred):
^
./armory/metrics/task.py:618:1: C901 'object_detection_AP_per_class' is too complex (20)
def object_detection_AP_per_class(
^
./armory/metrics/task.py:1139:1: C901 'apricot_patch_targeted_AP_per_class' is too complex (18)
def apricot_patch_targeted_AP_per_class(y_list, y_pred_list, iou_threshold=0.1):
^
./armory/metrics/task.py:1324:1: C901 'dapricot_patch_targeted_AP_per_class' is too complex (18)
def dapricot_patch_targeted_AP_per_class(y_list, y_pred_list, iou_threshold=0.1):
^
./armory/metrics/task.py:1722:5: C901 'GlobalHOTA.__call__' is too complex (12)
    def __call__(self, y_list, y_pred_list):
    ^
./armory/art_experimental/attacks/carla_mot_patch.py:292:5: C901 'CARLAMOTDapricotPatch.inner_generate' is too complex (19)
    def inner_generate(  # type: ignore
    ^
./armory/art_experimental/attacks/carla_obj_det_patch.py:302:5: C901 'CARLADapricotPatch.inner_generate' is too complex (20)
    def inner_generate(  # type: ignore
    ^
./armory/art_experimental/attacks/carla_obj_det_patch.py:616:5: C901 'CARLADapricotPatch.generate' is too complex (12)
    def generate(self, x, y=None, y_patch_metadata=None):
    ^
./armory/docker/images.py:147:1: C901 'ensure_image_present' is too complex (14)
def ensure_image_present(image_name: str) -> str:
^
./armory/instrument/config.py:191:5: C901 'ResultsLogWriter._write' is too complex (11)
    def _write(self, name, batch, result):
    ^
./armory/instrument/export.py:394:5: C901 'VideoTrackingExporter.get_sample' is too complex (11)
    def get_sample(self, x, with_boxes=False, y=None, y_pred=None):
    ^
./armory/utils/config_loading.py:83:1: C901 'load_model' is too complex (11)
def load_model(model_config):
^
./armory/utils/config_loading.py:252:1: C901 'load_label_targeter' is too complex (11)
def load_label_targeter(config):
^
./armory/scenarios/poisoning_witches_brew.py:102:5: C901 'WitchesBrewScenario._validate_attack_args' is too complex (26)
    def _validate_attack_args(self, adhoc_config, y_test):
    ^
./armory/scenarios/multimodal_so2sat_scenario.py:120:5: C901 'So2SatClassification.run_attack' is too complex (14)
    def run_attack(self):
    ^
./armory/scenarios/outputs.py:116:1: C901 'perturbation_accuracy' is too complex (12)
def perturbation_accuracy(benign, adversarial, perturbations, max_perturbation=None):
^
./armory/eval/evaluator.py:138:5: C901 'Evaluator.run' is too complex (17)
    def run(
    ^
./armory/data/datasets.py:103:5: C901 'ArmoryDataGenerator.get_batch' is too complex (18)
    def get_batch(self) -> Tuple[np.ndarray, Union[np.ndarray, List]]:
    ^
./armory/data/datasets.py:210:1: C901 '_parse_token' is too complex (11)
def _parse_token(token: str):
^
./armory/data/datasets.py:373:1: C901 '_generator_from_tfds' is too complex (33)
def _generator_from_tfds(
^
./armory/data/utils.py:34:1: C901 'maybe_download_weights_from_s3' is too complex (11)
def maybe_download_weights_from_s3(
^
./armory/data/utils.py:249:1: C901 'download_verify_dataset_cache' is too complex (24)
def download_verify_dataset_cache(dataset_dir, checksum_file, name):
^
25    C901 'run' is too complex (13)
25
~/git/davidslater/armory ~/armory
```